demo/pages/style/index.js

```diff
@@ -0,0 +1,53 @@
+Controller.$inject = ['$http', '$interval', 'qgrid'];
+export default function Controller($http, $interval, qgrid) {
+	this.model = qgrid.model();
+	this.rows = [];
+
+	$http.get('data/people/100.json')
+		.then(response => this.rows = response.data);
+
+	const randomColor = () => `${Math.floor(Math.random() * 16777215).toString(16)}`;
+	this.femaleColor = randomColor();
+	this.maleColor = randomColor();
+
+	this.model.style({
+		row: (row, dom) => {
```

Comment:
Its nor dom nor element just instance with one method class, and this is by design. Any ideas? Maybe add this method to context?